soccer/gameplay/play.py

```diff
@@ -32,3 +32,13 @@ def is_restart(cls):
     @classmethod
     def handles_goalie(cls):
         return False
+
+
+    ## Override to allow a play to run during the stopped state
+    # This is reserved only for very special plays that need to run during the
+    # stopped state.  Most plays should give up control during the stopped state
+    # to stopped.py, so stopped.py is the only play that should be overriding
+    # this.
+    @classmethod
+    def run_during_stopped(cls):
```

Comment:
I perfer functions just because I don't need to think about it too much (and it fit the rest of the style here). We could have also made this a boolean field, but (imo) it feels clunky to have things being called behind people's back (through getters/setters)